$NetBSD: patch-dpteng_device_cpp,v 1.1 2011/05/22 19:11:42 dholland Exp $

gcc apparently won't accept passing a packed field by reference
nowadays.

--- dpteng/device.cpp~	2002-08-13 15:02:21.000000000 +0000
+++ dpteng/device.cpp
@@ -4048,7 +4048,15 @@ if (bytesLeft >= bytesNeeded) {
 	// Return this object's manager SCSI ID instead of tag
       basic_P->attachedTo = myMgr_P()->getAddrL();
 	// Reverse the SCSI address bytes
-      reverseBytes(basic_P->attachedTo);
+      {
+	 /*
+	  * dholland 20110522 gcc apparently won't accept passing a
+	  * packed field by reference nowadays.
+	  */
+	 uLONG tmp = basic_P->attachedTo;
+	 reverseBytes(tmp);
+	 basic_P->attachedTo = tmp;
+      }
    }
      // Return the component list size
    fromEng_P->insert(compSize);
